Issue/16309 compact blogging prompt card

khaykov opened the pull request:
Fixes #16309

This PR ads a compact card to the Create Content bottom sheet. The are some placeholder logic for the answer button, but it is not wider to the button yet. Button toggles between Answer and Answered states for test purposes.

[![Image from Gyazo](https://i.gyazo.com/70a18ce6fa8aae4d54e7a64832a98942.png)](https://gyazo.com/70a18ce6fa8aae4d54e7a64832a98942)

To test:
- Enable BP FF and tap on the + icon on My Site screen.
- Confirm that the compact card is visible at the top of the bottom sheet, and that it looks like the design.
- With the BP FF off, confirm that the compact card is not visible in the bottom sheet.

## Regression Notes
1. Potential unintended areas of impact
- Card is visible when FF is off. 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- Atuomated and manual testing.

3. What automated tests I added (or what prevented me from doing so)
- FF test and card click action test.

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

khaykov committed:
Adding card to the actions adapter.

khaykov committed:
compact card layout and view holder

khaykov committed:
WIP VH

khaykov committed:
Updated view holders and adapter.

khaykov committed:
Add ff flag to view model. Added tests.

khaykov committed:
Fixed click method.

khaykov committed:
Added action to main activity.

khaykov committed:
fixed ktlint

khaykov committed:
Fixed some formatting issues.

RenanLukas commented on WordPress/src/main/java/org/wordpress/android/ui/main/MainActionListItem.kt:
```diff
@@ -10,7 +11,8 @@ sealed class MainActionListItem {
         CREATE_NEW_POST,
-        CREATE_NEW_STORY
+        CREATE_NEW_STORY,
+        ANSWER_BLOGGING_PROMP
```
Nitpick: I think there was a small typo here (`ANSWER_BLOGGING_PROMP` -> `ANSWER_BLOGGING_PROMPT`)

RenanLukas reviewed (approved):
Thanks for the PR @khaykov ! It works as expected :shipit:

khaykov committed:
Centered title and icon, updated margins.

khaykov committed:
Fixed typo.

khaykov commented:
@RenanLukas Thanks for the review! As you also mentioned in Slack, I fixed the centering of label + icon, and adjusted margins to be more visually close with the figma, at least until we get some final clarification on it.

khaykov committed:
Updated answered button margins as well.
